front-end/src/components/Themes.js

```diff
@@ -0,0 +1,175 @@
+import { Typography, Grid } from '@material-ui/core';
+import { createMuiTheme, withStyles } from "@material-ui/core/styles";
+import { Button } from '@material-ui/core';
+import { AccountCircle, Lock } from '@material-ui/icons';
+import background from './../assets/image/background.png';
+
+const theme = createMuiTheme({
+    typography: {
+      fontFamily: [
+        'Roboto',
+        '"Helvetica Neue"',
+        'Arial',
+        'sans-serif'
+      ].join(','),
+    },
+    Button: {
+      backgroundColor: '#227B3C', color: '#fff'
+    }
+});
+
+const GreenTextTypography = withStyles({
+    root: {
+        fontSize: '72px',
+        color: '#227B3C',
+    }
+})(Typography);
+
+const GreenSmallTypography = withStyles({
+    root: {
+        fontSize: '15px',
+        color: '#227B3C',
+    }
+})(Typography);
+
+const GreenSmallTypographyCreate = withStyles({
+    root: {
+        fontSize: '15px',
+        color: '#1261A3',
+    }
+})(Typography);
+
+const ValidatiionPasswordString = withStyles({
```

Comment:
This is only used in the Login component. As it is not shared with other files this code should live in Login.js.

There are several exports in this file that are not used in multiple files. If it is not used in multiple files than it should not be abstracted, please leave it with the componet/code that uses it.

Reply:
This theme file should only contain theme information that is used by multiple components.